frontend/src/Xbdd.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { Suspense, FC } from 'react';
+import { useSelector } from 'react-redux';
+import { ThemeProvider } from '@material-ui/core';
+import { BrowserRouter as Router, Switch, Route, useParams } from 'react-router-dom';
+
+import Navbar from 'modules/navbar/Navbar';
+import SummaryContainer from 'modules/summary/SummaryContainer';
+import ReportContainer from 'modules/report/ReportContainer';
+import ErrorBoundary from 'modules/errorBoundary/ErrorBoundary';
+import theme from 'AppTheme';
+import NotificationsView from 'modules/notifications/NotificationsView';
+
+import './Xbdd.css';
+
+import { RootStore } from 'rootReducer';
+
+const ReportPage: FC<{}> = () => {
```

Comment:
If you just use FC it will default to FC<{}>

Reply:
cool, will do